public/index.html

```diff
@@ -14,15 +14,15 @@
         </button>
         <div class="expandedProject">
           <div class="projectPictureBox">
-            <img class="projectPicture" src="{{imgSrc}}"></img>
+            <!--<img class="projectPicture" src="{{imgSrc}}"></img>-->
           </div>
           <div class='projectTextBox'>
             <p>{{description}}</p>
-            <h4 class="completedDate">{{dateCompleted}}</h4>
-            <h5 class="collaborators">{{colaborators}}</h5>
+            <h4 class="completedDate">{{updated_at}}</h4>
+            <a href="{{collaborators_url}}"<h5 class="collaborators">Collaborators</h5></a>
```

Comment:
`collaborators_url` is an API URL. (Strictly speaking, GitHub would be more RESTful if they served their API from the same URLs as people use.)

Try `{{html_url}}graphs/contributors` instead.